docs/.vitepress/config.ts

```diff
@@ -456,6 +456,20 @@ export default defineConfig({
         },
       }),
     ],
+    environments: {
+      client: {
+        dev: {
+          optimizeDeps: {
+            include: [
+              '@shikijs/vitepress-twoslash/client',
+              'gsap',
+              'gsap/dist/ScrollTrigger',
+              'gsap/dist/MotionPathPlugin',
+            ],
+          },
+        },
+      },
+    },
```

Comment:
If we use `dev.optimizeDeps` directly, these deps are also optimized for the `ssr` environment. I think we will need to move the ssr optimize step (only for this default one?) to be lazy on the first transform call if we want to avoid this.